tiling.js

```diff
@@ -2388,7 +2388,7 @@ export const Spaces = class Spaces extends Map {
 
         if (move && this.selectedSpace.selectedWindow) {
             const navigator = Navigator.getNavigator();
-            if (navigator._moving === null ||
+            if (navigator._moving == null ||
```

Comment:
Can you also add a comment here that this needs to be `==` instead of `===`? Otherwise we might break it again in the future on accident.

Reply:
Yeah, good idea - I'll stick a comment in develop.

Reply:
Actually, a much better (cleaner, simpler to avoid confusion) is just to use

```
            if (!navigator._moving ||

```

like we do here: https://github.com/paperwm/PaperWM/blob/4ee4d466047972636d7a7dbb3b943f6598fa4370/tiling.js#L4321

Reply:
Line 4321 is from the `takeWindow` function which (which the function in question calls) and actually does this same check (to create the `_moving` array).

I'll make this change and re-test and get it in develop to avoid future occurrences of this issue.